util/src/main/java/org/killbill/billing/util/glue/RedisShiroManagerProvider.java

```diff
@@ -32,17 +33,20 @@
 
 public class RedisShiroManagerProvider implements Provider<RedisShiroManager> {
 
-    private final CacheManager eh107CacheManager;
     private final SecurityManager securityManager;
+    private final SubjectDAO subjectDAO;
+    private final CacheManager eh107CacheManager;
     private final MetricRegistry metricRegistry;
     private final RedissonClient redissonClient;
 
     @Inject
     public RedisShiroManagerProvider(final SecurityManager securityManager,
+                                     final SubjectDAO subjectDAO,
```

Comment:
Do we support storing our sessions into Redis -- instead of database?

Reply:
Yup 😎